x-pack/plugins/observability/server/saved_objects/threshold.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { fold } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+import type { SavedObject, SavedObjectsType } from '@kbn/core/server';
+import { metricsExplorerViewSavedObjectRT } from '../types';
+
+export const thresholdExplorerViewSavedObjectName = 'threshold-explorer-view';
+
+const getThresholdExplorerViewTitle = (savedObject: SavedObject<unknown>) =>
+  pipe(
+    metricsExplorerViewSavedObjectRT.decode(savedObject),
+    fold(
+      () => `Threshold explorer view [id=${savedObject.id}]`,
+      ({ attributes: { name } }) => name
+    )
+  );
+
+export const threshold: SavedObjectsType = {
+  name: thresholdExplorerViewSavedObjectName,
+  hidden: false,
+  namespaceType: 'single',
```

Comment:
Are we sure this new type should go to the default (`.kibana`) index?

It seems to be the case, given the other so types from this plugin (`slo` and `composite-slo`) lives in there, but just want to confirm before approving.

Reply:
@pgayvallet, that's correct.